.github/workflows/ci-testing.yml

```diff
@@ -37,10 +37,6 @@ jobs:
             python-version: 3.8
             requires: 'latest'
             topic: 'text'
-          - os: ubuntu-20.04
-            python-version: 3.8
-            requires: 'latest'
-            topic: 'image_style_transfer'
```

Comment:
any reason?

Reply:
I've removed the "image_style_transfer" extras, they are now in "image" so we don't need seperate CI anymore